index.js

```diff
@@ -19,8 +19,9 @@ const format = (markup, url) => {
   const method = title.split('.').pop();
   const methodWithoutParens = method.replace(/\(\)/, '');
   const description = $('#wikiArticle > p')
-    .first()
-    .text()
+    .filter((index, element) => Boolean($(element).text())) // filter empty paragraphs
+    .map((index, element) => $(element).text())
```

Comment:
Hmm, now you mention it, it does seem unnecessary to map all the element's text content since we only need one. Not sure at all why I did in the first place, good catch :)